<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VIRATGeovis</class>
 <widget class="QMainWindow" name="VIRATGeovis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIRAT Geovis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons/icons.qrc">
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QVTKWidget" name="GeoView" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>951</width>
          <height>491</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QVTKWidget" name="map3d" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>971</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Query_Results"/>
    <addaction name="actionOpenXMLFile"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenXMLFile"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionFirst"/>
   <addaction name="actionBack"/>
   <addaction name="actionPause"/>
   <addaction name="actionPlay"/>
   <addaction name="actionForward"/>
   <addaction name="actionLast"/>
   <addaction name="actionSnap"/>
   <addaction name="separator"/>
   <addaction name="actionCheckTraces"/>
   <addaction name="actionDialogQueries"/>
  </widget>
  <action name="actionOpenXMLFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>OpenXML File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/pqVcrPlay24.png</normaloff>:/Icons/pqVcrPlay24.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play video</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/pqVcrPause24.png</normaloff>:/Icons/pqVcrPause24.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/pqVcrBack24.png</normaloff>:/Icons/pqVcrBack24.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/pqVcrFirst24.png</normaloff>:/Icons/pqVcrFirst24.png</iconset>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/pqVcrForward24.png</normaloff>:/Icons/pqVcrForward24.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/pqVcrLast24.png</normaloff>:/Icons/pqVcrLast24.png</iconset>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </action>
  <action name="actionSnap">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Snap</string>
   </property>
  </action>
  <action name="actionCheckTraces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>CheckTraces</string>
   </property>
  </action>
  <action name="actionDialogQueries">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>DialogQueries</string>
   </property>
  </action>
  <action name="actionOpen_Query_Results">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open Query Results...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
